app/controllers/users_controller.rb

```diff
@@ -116,8 +116,7 @@ def profile
                        .page(params[:page])
                        .order("nid DESC")
                        .where(status: 1, uid: @user.uid)
-    @questions = DrupalNode.questions
-                           .where(status: 1, uid: @user.uid)
+    @questions = @user.user.questions
```

Comment:
You're right -- this line (https://github.com/publiclab/plots2/pull/945/files#diff-4e05ad0d64e6100656b63ad1e78f32c5R112) shows that we're actually using a DrupalUsers record. Hmm. 

Oh, i get it! I just added a `legacy_user` fixture -- that has no corresponding User record, to test some other code. The test is:

https://github.com/publiclab/plots2/blob/master/test/functional/users_controller_test.rb#L81

```ruby
    get :profile, id: DrupalUsers.where(status: 1).last.name
```

So it's getting that legacy user fixture. Change `last` to `first` and it should start to work!